.github/workflows/cicd-deploy-to-dev.yml

```diff
@@ -14,8 +14,13 @@ env:
   IMAGE_TAG: ${{ github.sha }}
   ECR_REGISTRY: ${{ vars.DEVElOPMENT_ECR_REGISTRY }}
   ECR_REPOSITORY: ${{ vars.DEVElOPMENT_ECR_REPOSITORY }}
+  ADMIN_GITHUB_TOKEN: ${{ secrets.DEV_ADMIN_GITHUB_TOKEN }}
```

Comment:
Where is this token generated? Shouldn't this be a bot account PAT following the naming convention for these?

Reply:
Good spot, the token has not been generated yet so removed the reference for now 😁